### ISSUE TITLE
Progress bars work as they should in cmd or powershell

### ISSUE THREAD

**Issue by FaberoneX** [rsalmei/alive-progress#97]
Hi, I don't know if I'm doing something wrong, the progress bars worked flawlessly in the vscode terminal but when I tried with cmd or powershell they would just refuse to work. If I missed something in the documentation I apologize in advance.

This is how showtime looks, it keeps scrolling down indefinitely:

![2021_07_28_22_33_59_Windows_PowerShell](https://user-images.githubusercontent.com/15165439/127392020-12e145a4-e854-47a6-ae9a-0f5a15147ce2.png)

This is how a progress bar looks, there was no scrolling but still not right:

![2021_07_28_22_41_05_Windows_PowerShell_2](https://user-images.githubusercontent.com/15165439/127393181-9b459a49-c8fc-49bb-9595-525bfda471cd.png)

This is how it looks on vscode with the same exact code, the text updates correctly as it iterates, looks perfect:

![vscode_terminal](https://user-images.githubusercontent.com/15165439/127393209-e2fea38c-fceb-46d7-a9db-d60afbddf0d3.png)

I'm using Python 3.7.0, alive_progress v1.6.2 on Windows 10
Any help would be greatly appreciated.

**Comment by TheTechRobo** [comment]
Can you try it with the unreleased 2.0 version?

pull-request #51

**Comment by rsalmei** [comment]
Hey @FaberoneX,

It seems to me you're missing a sleep.
Please try to include at the top `import time`, and the inside the bar context: `time.sleep(.01)`.
I think this will improve considerably what you're seeing.

Then, regarding the scrolling, it is the same issue as not clearing the line:
![image](https://user-images.githubusercontent.com/6652853/127444792-c16b7fe6-23ef-4583-8140-fe4aa448e307.png)

The arrow indicates the last char I print in the bar. Then comes an ANSI Escape Code telling the terminal to clear to the end of the line...

Your terminal also clearly shows in showtime these control codes:
![image](https://user-images.githubusercontent.com/6652853/127446449-f55dc349-8fb3-4e79-abf7-4726ecc8324e.png)

That code is telling the terminal to "go up 41 lines", which would refresh all bar lines, but it just print it out, without interpreting it.
This is a Windows error, I have some other issue that handled that in #83, you must insert:
```python
import os
os.system("")
```

**Comment by FaberoneX** [comment]
@TheTechRobo 

Thank you, this worked perfectly, now the bars look good. FYI I had an error when I tried installing the branch, I fixed it by replacing readme.md's content with the letter a. Here's the error:

![2021_07_29_08_51_39_C_WINDOWS_system32_cmd](https://user-images.githubusercontent.com/15165439/127455493-b61093aa-2841-4b56-b337-eff5644464d0.png)

<hr>

@rsalmei 

Hi, thanks. I rolled back to master and tried what you said, adding os.system("") fixed the bars, now it works flawlessly, even without any sleep.
